packages/app/pages/today.tsx

```diff
@@ -0,0 +1,95 @@
+import { Avatar, Button, Card, Col, PageHeader, Row, Result } from "antd";
+import useSWR from "swr";
+import Schedule from "./schedule";
+import { Queue, QueuePartial } from "../../common/index";
+import { API } from "@template/api-client";
+import styled from "styled-components";
+
+const Navbar = () => {
+  return (
+    <PageHeader title={"Khoury Office Hours"}>
+      <h1>CS 2500</h1>
+    </PageHeader>
+  );
+};
+
+type QueueCardProps = {
+  queue: QueuePartial;
+};
+
+const PaddedCard = styled(Card)`
+  margin-bottom: 25px;
+`;
+
+const AvatarContainer = styled.div`
+  padding-left: 25px;
+  padding-right: 25px;
+  float: left;
+`;
+
+const CreateQueueButton = styled(Button)`
+  bottom: 0%;
+  width: 20%;
```

Comment:
We generally don't want to make widths percentages because it makes things really funky on small screen sizes. This can be changed to a fixed width or just not specified width is also fine.